### ISSUE TITLE
Cannot add alt tags for all images - header images, articles feature images

### ISSUE THREAD

**Issue by hlflanagan** [wid-tech/website#25]
Even with the WCAG template, not all images are properly tagged

**Comment by pamelatech** [comment]
Hey @fhobswid can you provide an example of this so Michael can reproduce for his team?

**Comment by mcresilkstart** [comment]
@fhobswid I have asked my techies how to add the alt-tags to these images and once they clarify to me I will pass the details on to you.

**Comment by mcresilkstart** [comment]
@fhobswid so header banners (static) aren't technically images so we can't add alt-tags to these.  FOr slide shows there is an alt-tag field when adding/editing slides.  TO add alt-tags to images in articles we would have to add that capability as a code change - I will raise an enhancement request for that.

**Comment by EndlessTrax** [comment]
@mcresilkstart checked on the status of this offline. His email reply:

```
Issue 25 - this is now fixed.  You can now add alt-tags to article images
```
